settings.gradle

```diff
@@ -65,6 +66,8 @@ project(':opencensus-exporter-stats-stackdriver').projectDir =
         "$rootDir/exporters/stats/stackdriver" as File
 project(':opencensus-exporter-stats-prometheus').projectDir =
         "$rootDir/exporters/stats/prometheus" as File
+project(':opencensus-exporter-trace-datadog').projectDir =
```

Comment:
Please move this to "Java8Compatible" projects (https://github.com/census-instrumentation/opencensus-java/pull/1652/files#diff-88b7c47e47b8ee65263b6784b86fa0a7R84).